lib/commands/search.jsx

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const { h, render } = require('ink')
+const { Search } = require('../components/search.jsx')
+
+const SearchCommand = module.exports = {
+  command: 'search',
+  aliases: ['s', 'se', 'find'],
+  describe: 'Search the NPM registry',
+  builder (y) {
+    return y.help().alias('help', 'h')
+      .options(SearchCommand.options)
+  },
+  options: Object.assign(require('../common-opts.js'), {}),
+  handler: search
+}
+
+async function search (argv) {
+  let unmount
+
+  const onError = () => {
+    unmount()
+    process.exit(1)
```

Comment:
is this the expected pattern for `ink`? It's so strange.

Reply:
@zkat I actually borrowed this pretty much verbatim from the `emma-cli`:

https://github.com/maticzav/emma-cli/blob/master/src/index.js#L51

Definitely a little weird, but I think it makes sense...